udata_search_service/config.py

```diff
@@ -6,9 +6,7 @@ class Config:
 
     ELASTICSEARCH_URL = os.environ.get('ELASTICSEARCH_URL') or 'http://localhost:9200'
 
-    DATASET_CATALOG_URL = 'https://www.data.gouv.fr/fr/datasets/r/f868cca6-8da1-4369-a78d-47463f19a9a3'
-    ORG_CATALOG_URL = 'https://www.data.gouv.fr/fr/datasets/r/b7bbfedc-2448-4135-a6c7-104548d396e7'
-    REUSE_CATALOG_URL = 'https://www.data.gouv.fr/fr/datasets/r/970aafa0-3778-4d8b-b9d1-de937525e379'
+    ELASTICSEARCH_INDEX_PREFIX = os.environ.get('ELASTICSEARCH_INDEX_PREFIX') or 'udata'
```

Comment:
Maybe we can use a more generic name for this, since it will have to be consistent:
- across services (udata, search...)
- across middlewares (Kafka, ES...)

Reply:
Hmm, this setting will only be used in udata-search-service as is.

The setting used previously in udata was `ELASTICSEARCH_INDEX_BASENAME`.
In our infra, it was set to `udata__instance_name` in ansible configuration.

Did you have a different logic in mind?

Reply:
I think if the setting is the same across all services it would be easier to reason about.

Reply:
Used UDATA_INSTANCE_NAME instead of ELASTICSEARCH_INDEX_PREFIX.
I find it less explicit about its implication though. I could also do:
```
ELASTICSEARCH_INDEX_PREFIX = os.environ.get('ELASTICSEARCH_INDEX_PREFIX', UDATA_INSTANCE_NAME)
```
and use ELASTICSEARCH_INDEX_PREFIX around. Not sure it would be less confusing though.
What did you have in mind?